### ISSUE TITLE
Drawing Test "07) Text Underlining" is broken on the PDF backend

### ISSUE THREAD

**Issue by slyrus** [McCLIM/McCLIM#562]
When running the "07) Text Underlining" from the clim-demo drawing tests test one gets:

```
Control stack exhausted (no more space for function call frames).
This is probably due to heavily nested or infinitely recursive function
calls, or a tail call that SBCL cannot or has not optimized away.

PROCEED WITH CAUTION.
   [Condition of type SB-KERNEL::CONTROL-STACK-EXHAUSTED]

Restarts:
 0: [SKIP] skip 07) Text Underlining
 1: [ABORT] Return to application command loop
 2: [ABORT] Return to application command loop
 3: [RETRY] Retry SLIME REPL evaluation request.
 4: [*ABORT] Return to SLIME's top level.
 5: [ABORT] abort thread (#<THREAD "repl-thread" RUNNING {1009018603}>)

Backtrace:
  0: (SB-KERNEL::CONTROL-STACK-EXHAUSTED-ERROR)
  1: ("foreign function: call_into_lisp")
  2: ("foreign function: post_signal_tramp")
  3: (CLIM-POSTSCRIPT-FONT::PSFONT-TEXT-EXTENTS :INVALID-VALUE-FOR-UNESCAPED-REGISTER-STORAGE :INVALID-VALUE-FOR-UNESCAPED-REGISTER-STORAGE) [more]
  4: (CLIM-POSTSCRIPT-FONT::PSFONT-TEXT-EXTENTS "Helvetica" "" :START 0 :END NIL)
  5: (CLIM-POSTSCRIPT-FONT::PSFONT-TEXT-EXTENTS "Helvetica" "" :START 0 :END NIL)
  6: (CLIM-POSTSCRIPT-FONT::PSFONT-TEXT-EXTENTS "Helvetica" "" :START 0 :END NIL)
  7: (CLIM-POSTSCRIPT-FONT::PSFONT-TEXT-EXTENTS "Helvetica" "" :START 0 :END NIL)
...
```
I don't remember this happening in the past, but it's possible it has always been here.

**Comment by slyrus** [comment]
Perhaps this is the source of the error:

https://github.com/McCLIM/McCLIM/commit/ec0748e91ca0b345c95e7cb3a8cc58103e9cd36b

**Comment by dkochmanski** [comment]
Fixed by #563.
